raiden/network/rpc/smartcontract_proxy.py

```diff
@@ -48,7 +48,8 @@ def __init__(
             contract_address: Address,
             call_function: Callable,
             transact_function: Callable,
```

Comment:
Why remove the Optiona() from here?

Reply:
When setting a parameters default to `None`, `mypy` infers that the type is `Optional`.